backend/python/app/tests/graphql/queries/test_story_query.py

```diff
@@ -0,0 +1,81 @@
+def test_stories(app, db, client):
+    result = client.execute(
+        """
+        {
+            stories {
+                id
+                title
+                description
+                contents  {
+                    id
+                }      
+            }
+        }
+    """
+    )
+
+    assert result == {"data": {"stories": []}}
+
+
+def test_story_by_id(db, client):
+    pass
+
+
+def test_story_by_id_invalid_id(db, client):
+    pass
+
+
+def test_stories_available_for_translation(db, client):
+    pass
+
+
+def test_stories_available_for_translation_invalid_language_string(db, client):
+    pass
+
+
+def test_stories_available_for_translation_negative_or_too_large_level(db, client):
+    pass
+
+
+def test_stories_available_for_translation_invalid_language_and_level_for_given_user(
+    db, client
+):
+    pass
+
+
+def test_stories_available_for_translation_user_not_approved(db, client):
+    pass
+
+
+def test_story_translaiton_by_id(db, client):
+    pass
+
+
+def test_story_translation_by_id_invalid_id(db, client):
+    pass
+
+
+def test_stories_available_for_review(db, client):
+    pass
+
+
+def test_stories_available_for_review_invalid_language_string(db, client):
+    pass
+
+
+def test_stories_available_for_review_negative_or_too_large_level(db, client):
+    pass
+
+
+def test_stories_available_for_review_invalid_language_and_level_for_given_user(
+    db, client
+):
+    pass
+
+
+def test_stories_available_for_review_user_not_approved(db, client):
+    pass
+
+
+def test_not_authenticated_for_endpoint(db, client):
```

Comment:
hmm perhaps there should be a test file for gql middleware rather than this test in `test_story_query.py`. It can test a gql request using `@require_authorization_by_role_gql` . Thoughts?

Reply:
Yeah I like this, can we eliminate all permissions tests then?

Reply:
> can we eliminate all permissions tests
All that use fixtures, yes. I think some mutations like assignReviewer throw an error if "permissions" arent met (ie user is not qualified to be reviewer)